src/utils.js

```diff
@@ -5,6 +5,12 @@ function buildMap(rows) {
   }, {});
 }
 
+function warnDeprecated(warning) {
+  if (process.env.NODE_ENV === 'development') {
+    console.warn(`DEPRECATED:  ${warning}`);
```

Comment:
Not totally sure how I feel about prefixing "DEPRECATED" in front of every message because the wording of some of the messages might make this is a bit awkward. Perhaps something like "Deprecation Notice:" would be better. What do you think?

Reply:
Yeah thats fine, I'll make the change.